src/main/menu.file.ts

```diff
@@ -1,39 +1,60 @@
+import { BrowserWindow, MenuItemConstructorOptions } from "electron";
+
 import { I18nService } from "jslib-common/abstractions/i18n.service";
 import { MessagingService } from "jslib-common/abstractions/messaging.service";
 
-import { isMacAppStore } from "jslib-electron/utils";
-
 import { IMenubarMenu } from "./menubar";
 
-import { MenuItemConstructorOptions } from "electron";
+import { FirstMenu } from "./menu.first";
+import { MenuAccount } from "./menu.updater";
+
+import { UpdaterMain } from "jslib-electron/updater.main";
+import { isMac, isMacAppStore } from "jslib-electron/utils";
 
-export class FileMenu implements IMenubarMenu {
+export class FileMenu extends FirstMenu implements IMenubarMenu {
   readonly id: string = "fileMenu";
 
   get label(): string {
     return this.localize("file");
   }
 
   get items(): MenuItemConstructorOptions[] {
-    return [
+    let items = [
       this.addNewLogin,
       this.addNewItem,
       this.addNewFolder,
       this.separator,
       this.syncVault,
       this.exportVault,
-      this.quitBitwarden,
     ];
-  }
 
-  private readonly _i18nService: I18nService;
-  private readonly _messagingService: MessagingService;
-  private readonly _isLocked: boolean;
+    if (!isMac()) {
+      items = [
+        ...items,
+        ...[
+          this.separator,
+          this.settings,
+          this.lock,
+          this.lockAll,
```

Comment:
I don't really like that we have lock in `file` to begin with. It feels like it should belong to Account especially if we add more of them.